io/io/src/TBufferFile.cxx

```diff
@@ -2275,7 +2291,7 @@ Int_t TBufferFile::WriteFastArray(void **start, const TClass *cl, Int_t n,
       (*streamer)(*this,(void*)start,0);
       return 0;
    }
-
+   if (n <= 0) return -1;
```

Comment:
@pcanal is there a way here to check the size of the class?

Reply:
We can get the size in memory but it is only vaguely related to the streamed size (however by definition the class streaming will be call other low level streaming function, so we don't 'need' it here)

Reply:
Ok, let's leave it then, thks!